dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/IGateway.cs

```diff
@@ -1,18 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // IGateway.cs
+using Grpc.Core;
 using Microsoft.AutoGen.Contracts;
+using Microsoft.AutoGen.Protobuf;
 
-namespace Microsoft.AutoGen.Runtime.Grpc.Abstractions;
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
 
 public interface IGateway : IGrainObserver
```

Comment:
Can we constrain the gateway to current functionality and add things back as we work through them

Reply:
Can you be more specific?

Reply:
The KV based state APIs should not be a part of the gateway if they are not on teh RPC

Reply:
No one is going to be using the gateway apis directly - if we are going to be restoring this functionality in another PR I'd rather not delete the code.